### ISSUE TITLE
Time series dataview buckets values "bug"

### ISSUE THREAD

**Issue by ivanmalagon** [CartoDB/Windshaft-cartodb#792]
### Context

We have an inconsistency in how Windshaft groups timestamps into bins and how UI needs / show them. 

### Steps to Reproduce

1. Import this dataset in Builder (rename to .csv) 
[dospuntos copy.txt](https://github.com/CartoDB/Windshaft-cartodb/files/1470566/dospuntos.copy.txt)
2. Create a time series widget
3. Set the widget timezone to `Azores` (UTC - 1) and aggregation to days.

We have two points with timestamps:
1. Soria `1970-01-02 00:00:00+00` Timestamp 86400
2. Zaragoza `1970-01-02 01:00:00+00` Timestamp 90000

The API returns 2 buckets

```
{
  "aggregation": "day",
  "offset": -3600,
  "timestamp_start": 0,
  "bin_width": 1800,
  "bins_count": 2,
  "bins_start": 3600,
  "nulls": 0,
  "bins": [
    {
      "bin": 0,
      "min": 86400,
      "max": 86400,
      "avg": 86400,
      "freq": 1,
      "timestamp": 3600
    },
    {
      "bin": 1,
      "min": 90000,
      "max": 90000,
      "avg": 90000,
      "freq": 1,
      "timestamp": 90000
    }
  ],
  "type": "histogram"
}
```

As you can see in the payload we have this info:
- Buckets start at `timestamp_start: 0` That is 1970-01-01 00:00:00
- First bucket has 1 point with the timestamp 86400. (1970-01-02 00:00:00)
- Second bucket has 1 point with the timestamp 90000. (1970-01-02 01:00:00) 

So if buckets start at 0 (day 1 00:00:00) and it includes 86400 (day 2 00:00:00) we have a problem. It includes a whole day **and a second** 😱 . 86400 should belong to the next bucket.

### Current Result

![bucketing bug](https://user-images.githubusercontent.com/1078228/32780709-946cde12-c942-11e7-8109-1bf9ac39b75f.gif)

As you can see in the GIF, we draw the two buckets returned by the API but when we filter we send whether day 1 (from 00:00:00 to 23:59:59) or day 2 (from 00:00:00 to 23:59:59)

Soria points was returned in the first bucket but when filtering from 86400 (day 2 00:00:00) it's included.

**Comment by Algunenano** [comment]
I'm still testing everything, but it should be solved with #804

**Comment by Algunenano** [comment]
@ivanmalagon I've tested and this appears to be fixed with the 4.3.0 release (already in production). Can you have a look please and let me know if there is any issue? Probably https://github.com/CartoDB/cartodb/issues/13070 can be closed too
